interpreter/token_objects/tobj.py

```diff
@@ -3,19 +3,25 @@
 
 
 class TObj(ABC):
-    __length = 0  # the number of tokens the object consists of
+    __length = 0  # !!! Unnecessary, should be deleted !!!
```

Comment:
If length is unnecesarry, then delete it. Be careful to delete it also where it was used.

Reply:
Alright, will delete it then